Fix precision of second-derivative coefficients

pbartholomew08 opened the pull request:
They were not using the correct precision, resulting in not
converging to machine zero in tests.

Thanks to Cédric for identifying issue and suggesting solution.

pbartholomew08 committed:
Fix precision of second-derivative coefficients

They were not using the correct precision, resulting in not
converging to machine zero in tests.

pbartholomew08 committed:
Fix precision of coefficients

mathrack commented on src/schemes.f90:
```diff
@@ -475,9 +475,9 @@ subroutine second_derivative(alsa1,as1,bs1,&
-     bsi  = (3./44. )/d2
+     alsai= two / eleven
+     asi  = (twelve / eleven)/d2
+     bsi  = (three / 44.0_mytype )/d2
```
Here, we can use `fortyfour` : https://github.com/xcompact3d/Incompact3d/blob/master/src/module_param.f90#L487

pbartholomew08 replied:
I'm personally opposed to this - see #66

pbartholomew08 committed:
Update dbg-scheme to confirm previous fix

Adds an RMS error computation for first and second derivatives in
x, y and z

Also updates the input.i3d to reflect changes in the namelists
